CMakeLists.txt

```diff
@@ -6,32 +6,36 @@ set(CMAKE_BUILD_TYPE Release)
 #set(CMAKE_CXX_COMPILER /usr/local/bin/g++-4.8)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/misc/")
-find_package(LibXml2 REQUIRED)
```

Comment:
Can you please leave LibXml2? It is needed on Linux when linking with EXIV2 (otherwise you'll get errors about missing symbols)